gvm/protocols/gmpv7.py

```diff
@@ -1499,35 +1499,288 @@ def help(self, **kwargs):
+                                   'nvt_selection')
+
+        cmd = XmlCommand('modify_config')
+        cmd.set_attribute('config_id', str(config_id))
+
+        if selection in 'nvt_pref':
+            _xmlpref = cmd.add_element('preference')
+            _xmlpref.add_element('nvt', attrs={'oid': nvt_oid})
+            _xmlpref.add_element('name', name)
+            _xmlpref.add_element('value', value)
+
+        elif selection in 'nvt_selection':
+            _xmlnvtsel = cmd.add_element('nvt_selection')
+            _xmlnvtsel.add_element('family', family)
+
+            if isinstance(nvt_oid, list):
+                for nvt in nvt_oid:
+                    _xmlnvtsel.add_element('nvt', attrs={'oid': nvt})
+            else:
+                _xmlnvtsel.add_element('nvt', attrs={'oid': nvt_oid})
+
+        elif selection in 'family_selection':
+            _xmlfamsel = cmd.add_element('family_selection')
+            _xmlfamsel.add_element('growing', '1')
+            _xmlfamily = _xmlfamsel.add_element('family')
+            _xmlfamily.add_element('name', family)
+            _xmlfamily.add_element('all', '1')
+            _xmlfamily.add_element('growing', '1')
+        else:
+            raise NotImplementedError
+
+        return self._send_xml_command(cmd)
+
+    def modify_credential(self, credential_id, name=None, comment=None,
+                          allow_insecure=None, certificate=None,
+                          key_phrase=None, private_key=None, login=None,
+                          password=None, auth_algorithm=None, community=None,
+                          privacy_algorithm=None, privacy_password=None,
+                          credential_type=None):
+        """Generates xml string for modify credential on gvmd.
+
+        Arguments:
+            name (str): Name of the new credential
+            comment (str, optional): Comment for the credential
+            allow_insecure (boolean, optional): Whether to allow insecure use of
+                 the credential
+            certificate (str, optional): Certificate for the credential
+            key_phrase (str, optional): Key passphrase for the private key
+            private_key (str, optional): Private key to use for login
+            login (str, optional): Username for the credential
+            password (str, optional): Password for the credential
+            auth_algorithm (str, optional): The auth_algorithm,
+                either md5 or sha1.
+            community (str, optional): The SNMP community
+            privacy_algorithm (str, optional): The SNMP privacy algorithm,
+                either aes or des.
+            privacy_password (str, optional): The SNMP privacy password
+            credential_type (str, optionla): The credential type. One of 'cc',
+            'snmp', 'up', 'usk'
+        """
+        if not credential_id:
+            raise RequiredArgument('modify_credential requires '
+                             'a credential_id attribute')
```

Comment:
This indention is wrong. The full string be either start with a four spaces indention on the next line or the second line need to be aligned to 'modify_credential